modules/libsodium/1.0.19/overlay/bzl/defs.bzl

```diff
@@ -0,0 +1,40 @@
+"""Rule for generating the version.h file for libsodium."""
+
+VERSION_H_TEMPLATE = "//:src/libsodium/include/sodium/version.h.in"
+DEFAULT_VERSION = "1.0.19"
```

Comment:
You can inject `module_version()` from the build file instead.

Reply:
Done